lsp.go

```diff
@@ -269,6 +286,21 @@ func (s *Server) sendErrorDiagnostics(ctx context.Context, currentFile lsp.URI,
 	return nil
 }
 
+func (s *Server) handleCompletion(
+	ctx context.Context,
+	reply jsonrpc2.Replier,
+	r jsonrpc2.Request,
+	log zerolog.Logger,
+) error {
```

Comment:
I suppose this is related to what you mentioned about having to support completion even when it is not exposed as a capability and then not implementing it would break stuff